src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1628,15 +1638,13 @@ def aks_check_acr(cmd, client, resource_group_name, name, acr):
         output = subprocess.check_output(
             cmd,
             universal_newlines=True,
+            stderr=subprocess.STDOUT,
         )
     except subprocess.CalledProcessError as err:
-        raise CLIError("Failed to check the ACR: {}".format(err))
+        raise CLIError("Failed to check the ACR: {} Command output: {}".format(err, err.output))
     if output:
-        print(output)
-        if os.getenv("PYTEST_CURRENT_TEST", None):
-            return output
-    else:
-        raise CLIError("Failed to check the ACR.")
+        return output
+    raise CLIError("Failed to check the ACR.")
```

Comment:
Could you please use the specific error types instead of `CLIError`?

Reply:
Sure, resolved in commit [9b74ec0](https://github.com/Azure/azure-cli/pull/19115/commits/9b74ec07d97b88cedb689a217ab22270428b74ff).